front-end/src/components/Post/VoteRefrendum.tsx

```diff
@@ -0,0 +1,178 @@
+	const { queueNotification } = useContext(NotificationContext);
+
+	type ConvictionType = 'Locked1x' | 'Locked2x' | 'Locked3x' | 'Locked4x' | 'Locked5x' | 'Locked6x';
+	const [conviction, setConviction] = useState<ConvictionType>('Locked1x');
+
+	const onConvictionChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+		const convictionValue = event.currentTarget.value as ConvictionType;
+		setConviction(convictionValue);
+	};
+
+	const voteRefrendum = async (aye: boolean) => {
+		if (!api) {
+			return;
+		}
+
+		const linkedAccount = await getLinkedAccount();
+
+		if (!linkedAccount) {
+			return;
+		}
+
+		const vote = api.tx.democracy.vote(referendumId || 0, { aye, conviction });
+
+		vote.signAndSend(linkedAccount.address, ({ status }) => {
+			if (status.isFinalized) {
+				queueNotification({
+					header: 'Success!',
+					message: `Completed at block hash #${status.asFinalized.toString()}`,
+					status: NotificationStatus.SUCCESS
+				});
+			} else {
+				queueNotification({
+					header: 'Info!',
+					message: `Current status: ${status.type}`,
+					status: NotificationStatus.WARNING
+				});
+			}
+		}).catch((error) => {
+			console.log(':( transaction failed');
+			console.error('ERROR:', error);
+			queueNotification({
+				header: 'Failed!',
+				message: error.message,
+				status: NotificationStatus.ERROR
+			});
+		});
+	};
+
+	return (
+		<div className={className}>
+			{/* <div className='card'>
+				<h2>Votes</h2>
+				<Divider/>
+				<Grid>
+					<Grid.Column width={6}>
+						<div><b>Total KSM Locked</b></div>
+						<div>241,547 KSM</div>
+					</Grid.Column>
+					<Grid.Column width={5}>
+						<div><b>Turnout</b></div>
+						<div>12,415%</div>
+					</Grid.Column>
+					<Grid.Column width={5}>
+						<div><b>Threshold</b></div>
+						<div>91.2%</div>
+					</Grid.Column>
+				</Grid>
+				<Divider/>
+				<Grid>
+					<Grid.Column width={6}>
+						<div><b>Total Votes</b></div>
+						<div>2,311,547</div>
+					</Grid.Column>
+					<Grid.Column width={5}>
+						<div><b>Yes</b></div>
+						<div>1,588,731</div>
+					</Grid.Column>
+					<Grid.Column width={5}>
+						<div><b>No</b></div>
+						<div>725,826</div>
+					</Grid.Column>
+				</Grid>
+			</div> */}
+			<div className='card'>
+				<h2>Your Vote</h2>
+				<Form standalone={false}>
+					<Form.Group>
+						<Form.Field width={16}>
+							<label>Vote Lock</label>
+							<select onChange={onConvictionChange}>
+								<option value='Locked1x'>2 weeks lock</option>
+								<option value='Locked2x'>4 weeks lock</option>
+								<option value='Locked3x'>6 weeks lock</option>
+								<option value='Locked4x'>8 weeks lock</option>
+								<option value='Locked5x'>10 weeks lock</option>
+							</select>
+							{/* <div>
+								120 KSM * 2 Lock periods = <b>240 votes</b>
+							</div> */}
+						</Form.Field>
+					</Form.Group>
+					<Form.Group>
+						<Form.Field width={8}>
+							<label>&nbsp;</label>
```

Comment:
Do we need a label here? having a space in there is hacky :) let's add a <br/> or a margin or anyting if you need space.

Reply:
removed.